src/concrete_minkowski_difference.jl

```diff
@@ -0,0 +1,76 @@
+export minkowski_difference, pontryagin_difference
+
+"""
+    minkowski_difference(P::LazySet{N}, Q::LazySet{N}) where {N<:Real}
+
+Concrete Minkowski difference (geometric difference) for a pair of
+convex sets.
+
+### Input
+
+- `P` -- polytopic set
+- `Q` -- compact convex set that is subtracted from `P`
+
+### Output
+
+An `HPolytope` that corresponds to the Minkowski difference of `P` minus `Q` if
+`P` is bounded, and an `HPolyhedron` if `P` is unbounded.
+
+### Notes
+
+This function requires that the list of constraints of the set `P` is
+available and that the set `Q` is bounded.
+
+### Algorithm
+
+This function implements theorems 2.3 in [1], which we state next.
```

Comment:
```suggestion
This function implements Theorem 2.3 in [1], which we state next.
```

Reply:
Alternative (as you prefer):
```suggestion
This function implements [1, Theorem 2.3], which we state next.
```